ui/src/app/cluster-workflow-templates/cluster-workflow-template-list.tsx

```diff
@@ -20,6 +20,7 @@ import {services} from '../shared/services';
 import {ClusterWorkflowTemplateCreator} from './cluster-workflow-template-creator';
 
 import './cluster-workflow-template-list.scss';
+import useTimestamp, {TIMESTAMP_KEYS} from '../shared/use-timestamp';
```

Comment:
Per https://github.com/argoproj/argo-workflows/pull/13160#discussion_r1633495520 and https://github.com/argoproj/argo-workflows/pull/13343#discussion_r1792535276, again, please follow the existing import style...

Reply:
I had to manually fix these issues in https://github.com/argoproj/argo-workflows/pull/13813 as they are unsortable by tooling since they were placed after a side effect import